client/assets/less/pages/place-detail.less

```diff
@@ -0,0 +1,105 @@
+.page-detail {
```

Comment:
I can not find this class in place-detail.template. Where is it use? And is it important?